### PR TITLE
Add Non-Hostile UPP ERT

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Adds a non-hostile UPP ERT alternative to the hostile ERT spawn. Has the same chance to spawn as the hostile version. 

# Explain why it's good for the game

In lore the UPP enjoy a cold war esque relationship with the UA, a possibly neutral ERT helps better reflect this.

Furthermore, as more UPP are added to the game, the need to help shift the community meta regarding UPP away from 100% hostile to 50/50 hostile is going to become more important, this would be 1 small step. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: UPP ERT's have a chance to be neutral to the USCM.
/:cl:

### CODE REVIEW

**Comment by JackieEstegado** [comment]
You may want to put in an exclusivity check so that a (non admin) UPP ERT of the oppossing friendliness cant spawn if one was spawned earlier (ie by an admin)

**Comment by fira** [comment]
Funny. A global toggle to ensure consistency might be cool but that means having to hack into ERT code

File: code/datums/emergency_calls/upp.dm [hunk 1]
```diff
@@ -106,6 +124,21 @@
 	max_engineers = 2
 	max_synths = 1
 	heavy_pick = FALSE
+	hostility = TRUE
+
+/datum/emergency_call/upp/platoon/New()
+	..()
+	arrival_message = "[MAIN_SHIP_NAME] t*is i* UP* d^sp^*ch`. STr*&e teaM, #*u are cLe*% for a*pr*%^h. Pr*mE a*l wE*p^ns and pR*epr# t% r@nd$r a(tD."
+	objectives = "Eliminate the UA Forces to ensure the UPP prescence in this sector is continued. Listen to your superior officers and take over the [MAIN_SHIP_NAME] at all costs."
```
Comment (harryob) [inline code/datums/emergency_calls/upp.dm:132]:
```suggestion
	objectives = "Eliminate the UA Forces to ensure the UPP presence in this sector is continued. Listen to your superior officers and take over the [MAIN_SHIP_NAME] at all costs."
```

File: code/datums/emergency_calls/upp.dm [hunk 2]
```diff
@@ -94,10 +96,26 @@
...
 
 
+/datum/emergency_call/upp/hostile
+	name = "UPP Naval Infantry (Squad) (Hostile)"
+	hostility = TRUE
+
+/datum/emergency_call/upp/hostile/New()
+	..()
+	arrival_message = "[MAIN_SHIP_NAME] t*is i* UP* d^sp^*ch`. STr*&e teaM, #*u are cLe*% for a*pr*%^h. Pr*mE a*l wE*p^ns and pR*epr# t% r@nd$r a(tD."
+	objectives = "Eliminate the UA Forces to ensure the UPP prescence in this sector is continued. Listen to your superior officers and take over the [MAIN_SHIP_NAME] at all costs."
```
Comment (harryob) [inline code/datums/emergency_calls/upp.dm:106]:
```suggestion
	objectives = "Eliminate the UA Forces to ensure the UPP presence in this sector is continued. Listen to your superior officers and take over the [MAIN_SHIP_NAME] at all costs."
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**